test/course_planner_web/controllers/user_controller_test.exs

```diff
@@ -206,6 +206,15 @@ defmodule CoursePlanner.UserControllerTest do
     assert Repo.get_by(User, email: "foo@bar.com")
   end
 
+  test "does not update password if is too short" do
+    user = insert(:student, name: "Foo", family_name: "Bar")
+    user_conn = guardian_login_html(user)
```

Comment:
You can simply `login_as(:student)`, it'll churn some code.

Reply:
yes but if you see i need the user as well. then afterward i have get the user from conn which makes it the same two lines :)

Reply:
Well, you can `%{current_user: user} = conn = login_as(:student)`.
I'll approve regardless since I'm being picky 😄

Reply:
nooice. sorry didn't see this coming :D

Reply:
done